frontend/packages/component-library/src/image/Image.tsx

```diff
@@ -0,0 +1,93 @@
+import classNames from 'classnames';
+import {CSSProperties, ImgHTMLAttributes} from 'react';
+
+import moduleStyles from './image.module.scss';
+
+export interface ImageProps extends ImgHTMLAttributes<HTMLElement> {
+  /** Image source */
+  src: string;
+  /** Image alt text */
+  altText?: string;
+  /** Image loading attribute */
+  loading?: 'eager' | 'lazy';
+  /** Image decoration */
+  decoration?: 'none' | 'border' | 'shadow';
+  /** Image container width. You can either pass it as a prop, or use custom className to set the needed width value.
+   * Using custom className is recommended, but we also support setting it via prop for the cases when it might be
+   * impossible to use custom className. */
+  width?: string | number;
+  /** Image container height. You can either pass it as a prop, or use custom className to set the needed height value.
+   Using custom className is recommended, but we also support setting it via prop for the cases when it might be
+   impossible to use custom className. */
+  height?: string | number;
+  /** Optional inline styles. You can either pass it as a prop, or use custom className to set the needed styles values.
+   Using custom className is recommended, but we also support setting it via prop for the cases when it might be
+   impossible to use custom className. */
+  style?: CSSProperties;
```

Comment:
Cool! ✨

Reply:
Oh, thanks for noticing this! I actually plan to remove this `style` and `height` and `width` from prop types :)  as I don't think we need this. However, inline styles will still be supported alongside with other native htmlelement props, meaning you won't see it in type definition and storybook, but there still will be an option to pass inline styles object prop if it's really required